src/main/java/net/snowflake/ingest/streaming/internal/FlushService.java

```diff
@@ -222,6 +214,25 @@ String getFilePath() {
         this.flushWorker);
   }
 
+  /** If tracing is enabled, print always else, check if it needs flush or is forceful. */
+  private void logFlushTask(boolean isForce, long timeDiffMillis) {
```

Comment:
Hi @sfc-gh-mnaides let me know if this sounds good! trace everything, if DEBUG, only during isNeedFlush and isForce..